.github/workflows/docker.yml

```
@@ -0,0 +1,65 @@
+name: Docker
+
+permissions:
+  packages: write
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
```

The formatting should match the other yaml files in the repo, so `-` not on its own line, and blank line between list items.

```suggestion
      - name: Checkout
        uses: actions/checkout@v2

      - name: Docker meta
```

---

Fixed.